pkg/controllers/provisioning/scheduling/topologygroup.go

```diff
@@ -91,8 +113,8 @@ func NewTopologyGroup(topologyType TopologyType, topologyKey string, pod *v1.Pod
 		rawSelector:  labelSelector,
 		nodeFilter:   nodeSelector,
```

Comment:
```suggestion
		nodeFilter:   nodeFilter,
```

nit: Consider calling this `nodeFilter` now since it's no longer just a nodeSelector